packages/oui-pagination/README.md

```diff
@@ -0,0 +1,125 @@
+# oui-pagination
+
+<component-status cx-design="partial" ux="prototype"></component-status>
+
+oui-pagination is a package which provide styles for the pagination component.
+
+## Installation
+
+```less
+@import 'oui-pagination/pagination';
+```
+
+## Usage
+
+### Button group
+
+```html:preview
+<div class="oui-pagination">
+  <oui-dropdown arrow>
+    <a tabindex="-1"
+        role="button"
+        class="oui-link"
+        oui-dropdown-trigger>
+        1 - 25</a>
+    &nbsp;
+    <oui-dropdown-content>
+      <div role="menu" class="oui-pagination-menu">
+        <div class="oui-pagination-menu__group">Results per page</div>
+        <div class="oui-pagination-menu__items-list">
+          <button type="button" class="oui-pagination-menu__item" disabled="disabled">25</button>
+          <button type="button" class="oui-pagination-menu__item">50</button>
+          <button type="button" class="oui-pagination-menu__item">100</button>
+          <button type="button" class="oui-pagination-menu__item">300</button>
+        </div>
+      </div>
+    </oui-dropdown-content>
+  </oui-dropdown>
+  <span>of 100 results</span>
+
+  <div class="oui-pagination__selector">
+    <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+      type="button"
+      aria-label="Previous page">
+      <i class="oui-icon oui-icon-chevron-left" aria-hidden="true"></i>
+    </button>
+    <div class="oui-button-group">
+      <button class="oui-button oui-button_primary oui-button_small-width oui-pagination-button_selected" disabled="disabled">1</button>
+      <button class="oui-button oui-button_secondary oui-button_small-width">2</button>
+      <button class="oui-button oui-button_secondary oui-button_small-width">3</button>
+      <button class="oui-button oui-button_secondary oui-button_small-width">4</button>
+      <button class="oui-button oui-button_secondary oui-button_small-width">5</button>
+    </div>
+    <div class="oui-select oui-select_inline"
+      ng-if="$ctrl.pageCount > 5">
+      <select class="oui-select__input"
+          ng-model="$ctrl.currentPage"
+          ng-change="$ctrl.setPageNumber(page)"
+          ng-options="page as $ctrl.capitalize($ctrl.config.words.page) + ' ' + page + ' ' + $ctrl.config.words.of + ' ' + $ctrl.pageCount for page in $ctrl.pagesList">
+      </select>
+      <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
+    </div>
+    <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+      type="button"
+      aria-label="Next page">
+        <i class="oui-icon oui-icon-chevron-right" aria-hidden="true"></i>
+      </button>
+  </div>
+</div>
+```
+
+### Select
+
+```html:preview
+<div class="oui-pagination">
+  <oui-dropdown arrow>
+    <a tabindex="-1"
+        role="button"
+        class="oui-link"
+        oui-dropdown-trigger>
+        1 - 25</a>
```

Comment:
On mobile this text is not on the right place. 

![image](https://user-images.githubusercontent.com/16029436/33718078-3a5193ce-db2a-11e7-9a86-7c6034e2c228.png)

The **1 - 25** should be at the left of **of 100 results** instead of being under.

Reply:
yes, fixed